app/controllers/plot_plants_controller.rb

```diff
@@ -0,0 +1,8 @@
+class PlotPlantsController < ApplicationController
+
+  def destroy
+    @plot_plant = PlotPlant.find(params[:id])
+    @plot_plant.destroy
+    redirect_to plots_path
+  end
```

Comment:
When you update your route and get both the plant and plot ID's, use that to find the record on the join table and delete the row.